.github/workflows/tests.yaml

```diff
@@ -27,6 +27,8 @@ jobs:
           - {name: Linux, python: '3.9', os: ubuntu-latest, tox: py39}
```

Comment:
I wonder if this one should become the 3.10 default, with 3.11-dev/3.9 being listed below.

cc @davidism

Reply:
I didn't want to do this here. If needed, I prefer if done in a follow-up PR.